<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  
  <link rel="canonical" href="http://biodatamodels.github.io/ontology-associations/all/">
  <link rel="shortcut icon" href="../img/favicon.ico">
  <title>ontology_associations schema - ontology association</title>
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700" />

  <link rel="stylesheet" href="../css/theme.css" />
  <link rel="stylesheet" href="../css/theme_extra.css" />
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/github.min.css" />
  
  <script>
    // Current page data
    var mkdocs_page_name = "ontology_associations schema";
    var mkdocs_page_input_path = "all.md";
    var mkdocs_page_url = "/ontology-associations/all/";
  </script>
  
  <script src="../js/jquery-2.1.1.min.js" defer></script>
  <script src="../js/modernizr-2.8.3.min.js" defer></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/highlight.min.js"></script>
  <script>hljs.initHighlightingOnLoad();</script> 
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
        <a href=".." class="icon icon-home"> ontology association</a>
        <div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
      <input type="text" name="q" placeholder="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../home/">Home</a>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="..">Index</a>
                    </li>
                </ul>
                <ul>
                    <li class="toctree-l1"><a class="reference internal" href="../credits/">Credits</a>
                    </li>
                </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="..">ontology association</a>
      </nav>

      
      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="..">Docs</a> &raquo;</li>
    
      
    
    <li>ontology_associations schema</li>
    <li class="wy-breadcrumbs-aside">
      
        <a href="https://github.com/biodatamodels/ontology-associations/edit/master/docs/all.md"
          class="icon icon-github"> Edit on GitHub</a>
      
    </li>
  </ul>
  
  <hr/>
</div>

          <div role="main">
            <div class="section">
              
                <h1 id="ontology_associations-schema">ontology_associations schema</h1>
<p>Various association data models</p>
<h3 id="classes">Classes</h3>
<ul>
<li><a href="../Annotation/">Annotation</a><ul>
<li><a href="../Association/">Association</a> - generic association between an entity and an ontological term<ul>
<li><a href="../GpadAssociation/">GpadAssociation</a> - line of GPAD</li>
<li><a href="../HumanPhenotypeOntologyAssociation/">HumanPhenotypeOntologyAssociation</a></li>
<li><a href="../PombasePhafAssociation/">PombasePhafAssociation</a> - line of PHAF</li>
<li><a href="../XafAssociation/">XafAssociation</a> - line of xAF. xAF is a generalization of GAF designed to work for other ontologies than GO, and for other entities beyond genes and proteins. xAF has the same column headers to GAF, but has fewer restrictions. <ul>
<li><a href="../GafAssociation/">GafAssociation</a> - line of GAF<ul>
<li><a href="../SimpleGafAssociation/">SimpleGafAssociation</a></li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
<li><a href="../Interaction/">Interaction</a> - An interaction between two molecular entities<ul>
<li><a href="../MitabInteraction/">MitabInteraction</a><ul>
<li><a href="../CausaltabInteraction/">CausaltabInteraction</a></li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
<li><a href="../ComplexDiseasePhenotypeOntologyAssociation/">ComplexDiseasePhenotypeOntologyAssociation</a></li>
<li><a href="../Document/">Document</a> - root class for association or entity documents<ul>
<li><a href="../AssociationDocument/">AssociationDocument</a><ul>
<li><a href="../ComplexDiseasePhenotypeOntologyAssociationDocument/">ComplexDiseasePhenotypeOntologyAssociationDocument</a></li>
<li><a href="../GafAssociationDocument/">GafAssociationDocument</a></li>
<li><a href="../GpadAssociationDocument/">GpadAssociationDocument</a></li>
<li><a href="../HumanPhenotypeOntologyAssociationDocument/">HumanPhenotypeOntologyAssociationDocument</a></li>
<li><a href="../PombasePhafAssociationDocument/">PombasePhafAssociationDocument</a></li>
</ul>
</li>
<li><a href="../GpiDocument/">GpiDocument</a> - A collection of gpi entities plus associated metadata</li>
</ul>
</li>
<li><a href="../Entity/">Entity</a><ul>
<li><a href="../BiologicalEntity/">BiologicalEntity</a><ul>
<li><a href="../GpiEntity/">GpiEntity</a> - An entity that can be annotated using GO. May be a gene, gene product, or complex</li>
</ul>
</li>
<li><a href="../ControlledTerm/">ControlledTerm</a><ul>
<li><a href="../OntologyClass/">OntologyClass</a><ul>
<li><a href="../Taxon/">Taxon</a></li>
</ul>
</li>
<li><a href="../RelationTerm/">RelationTerm</a></li>
</ul>
</li>
<li><a href="../Provider/">Provider</a></li>
<li><a href="../Publication/">Publication</a></li>
</ul>
</li>
<li><a href="../RelationalConstruct/">RelationalConstruct</a><ul>
<li><a href="../AnnotationExtension/">AnnotationExtension</a> - set of expressions all true</li>
<li><a href="../PropertyValuePair/">PropertyValuePair</a></li>
</ul>
</li>
</ul>
<h3 id="mixins">Mixins</h3>
<ul>
<li><a href="../Denormalized/">Denormalized</a> - mixin for a denormalized class. A denormalized class is formed from a join of two or more other classes</li>
<li><a href="../GoRelated/">GoRelated</a> - mixin for any association that is GO related</li>
<li><a href="../Normalized/">Normalized</a> - mixin for a normalized class, in the sense of database normal forms</li>
</ul>
<h3 id="slots">Slots</h3>
<ul>
<li><a href="../Alias%28es%29_interactor_A/">Alias(es) interactor A</a></li>
<li><a href="../Alias%28es%29_interactor_B/">Alias(es) interactor B</a></li>
<li><a href="../Allele_description/">Allele description</a> - The allele description specifically describes the change; see the Phenotype data bulk upload format for details on allele types and descriptions.</li>
<li><a href="../Allele_name/">Allele name</a> - Allele names are optional for upload, but are included where available in the download. Column 10 lists the preferred allele name, and any alternative names are in column 11.</li>
<li><a href="../Allele_synonym/">Allele synonym</a> - See note on Allele name above. Multiple entries are separated with pipes (|).</li>
<li><a href="../Allele_type/">Allele type</a> - Allowed allele types, example descriptions, etc. are shown in the table at the bottom of the <a href="https://www.pombase.org/documentation/phenotype-data-bulk-upload-format">Phenotype data bulk upload</a> format page.</li>
<li><a href="../Alt._ID%28s%29_interactor_A/">Alt. ID(s) interactor A</a></li>
<li><a href="../Alt._ID%28s%29_interactor_B/">Alt. ID(s) interactor B</a></li>
<li><a href="../Annotation%28s%29_interactor_A/">Annotation(s) interactor A</a></li>
<li><a href="../Annotation%28s%29_interactor_B/">Annotation(s) interactor B</a></li>
<li><a href="../Biological_Effect_interactor_A/">Biological Effect interactor A</a></li>
<li><a href="../Biological_Effect_interactor_B/">Biological Effect interactor B</a></li>
<li><a href="../Biological_role%28s%29_interactor_A/">Biological role(s) interactor A</a></li>
<li><a href="../Biological_role%28s%29_interactor_B/">Biological role(s) interactor B</a></li>
<li><a href="../Causal_Regulatory_Mechanism/">Causal Regulatory Mechanism</a></li>
<li><a href="../Causal_statement/">Causal statement</a></li>
<li><a href="../Checksum%28s%29_interactor_A/">Checksum(s) interactor A</a></li>
<li><a href="../Checksum%28s%29_interactor_B/">Checksum(s) interactor B</a></li>
<li><a href="../Condition/">Condition</a> - Conditions use a small ontology maintained in-house by PomBase curators, and we can help you interpret the IDs. Multiple entries are shown in cases where more than one condition detail applies at the same time (e.g. high temperature, minimal medium), and are separated with commas (,).</li>
<li><a href="../Confidence_value%28s%29/">Confidence value(s)</a></li>
<li><a href="../Creation_date/">Creation date</a></li>
<li><a href="../Expansion_method%28s%29/">Expansion method(s)</a></li>
<li><a href="../Experimental_role%28s%29_interactor_A/">Experimental role(s) interactor A</a></li>
<li><a href="../Experimental_role%28s%29_interactor_B/">Experimental role(s) interactor B</a></li>
<li><a href="../Expression/">Expression</a> - The Expression column contains one of these values: ‘overexpression’, ‘knockdown’, ‘endogenous’, ‘null’, ‘not specified’. Deletions always have ‘null’ expression.</li>
<li><a href="../Feature%28s%29_interactor_A/">Feature(s) interactor A</a></li>
<li><a href="../Feature%28s%29_interactor_B/">Feature(s) interactor B</a></li>
<li><a href="../Gene_name/">Gene name</a> - Gene names are optional for upload, but are included where available in the download.</li>
<li><a href="../Genotype_description/">Genotype description</a> - The Background genotype description column is for alleles in the background, such as selectable markers; these details are optional. To avoid redundancy, it does not repeat the allele of interest (from column 4 or 10) in the genotype column.</li>
<li><a href="../Host_organism%28s%29/">Host organism(s)</a></li>
<li><a href="../ID%28s%29_interactor_A/">ID(s) interactor A</a></li>
<li><a href="../ID%28s%29_interactor_B/">ID(s) interactor B</a></li>
<li><a href="../Identification_method_participant_A/">Identification method participant A</a></li>
<li><a href="../Identification_method_participant_B/">Identification method participant B</a></li>
<li><a href="../Interaction_Checksum%28s%29/">Interaction Checksum(s)</a></li>
<li><a href="../Interaction_Xref%28s%29/">Interaction Xref(s)</a></li>
<li><a href="../Interaction_annotation%28s%29/">Interaction annotation(s)</a></li>
<li><a href="../Interaction_detection_method%28s%29/">Interaction detection method(s)</a></li>
<li><a href="../Interaction_identifier%28s%29/">Interaction identifier(s)</a></li>
<li><a href="../Interaction_parameter%28s%29/">Interaction parameter(s)</a></li>
<li><a href="../Interaction_type%28s%29/">Interaction type(s)</a></li>
<li><a href="../Negative/">Negative</a></li>
<li><a href="../Parental_strain/">Parental strain</a> - The Parental strain column is for the parental strain designation, such as 972 h-, 975 h+, etc. This column must be filled in, but “unknown” is allowed.</li>
<li><a href="../Penetrance/">Penetrance</a> - Penetrance describes the proportion of a population that shows a cell-level phenotype. Penetrance data are represented as percents or entries from the in-house FYPO_EXT ontology (FYPO_EXT:0000001 = high; FYPO_EXT:0000002 = medium; FYPO_EXT:0000003 = low; FYPO_EXT:0000004 = full).</li>
<li><a href="../Publication_1st_author%28s%29/">Publication 1st author(s)</a></li>
<li><a href="../Publication_Identifier%28s%29/">Publication Identifier(s)</a></li>
<li><a href="../Severity/">Severity</a> - Severity (formerly designated “expressivity”) uses the FYPO_EXT ontology described in description of <a href="../Penetrance/">Penetrance</a></li>
<li><a href="../Source_database%28s%29/">Source database(s)</a></li>
<li><a href="../Stoichiometry%28s%29_interactor_A/">Stoichiometry(s) interactor A</a></li>
<li><a href="../Stoichiometry%28s%29_interactor_B/">Stoichiometry(s) interactor B</a></li>
<li><a href="../Strain_name_%28background%29/">Strain name (background)</a> - The Background strain name column is used for a lab’s in-house name/ID/designation for the background strain (i.e. the derivative of the parental strain that has selectable marker alleles etc.). The description in the Genotype column should match this background strain.</li>
<li><a href="../Taxid_interactor_A/">Taxid interactor A</a></li>
<li><a href="../Taxid_interactor_B/">Taxid interactor B</a></li>
<li><a href="../Type%28s%29_interactor_A/">Type(s) interactor A</a></li>
<li><a href="../Type%28s%29_interactor_B/">Type(s) interactor B</a></li>
<li><a href="../Update_date/">Update date</a></li>
<li><a href="../Xref%28s%29_interactor_A/">Xref(s) interactor A</a></li>
<li><a href="../Xref%28s%29_interactor_B/">Xref(s) interactor B</a></li>
<li><a href="../annotation_date/">annotation date</a> - date when annotation was made<ul>
<li><a href="Date.md">Date</a> - The date is the date on which the annotations are created. Format: YYYY-MM-DD</li>
</ul>
</li>
<li><a href="../annotation_extensions/">annotation extensions</a> - disjunction<ul>
<li><a href="../Extension/">Extension</a> - The Extension column is used to record when a mutation in one gene affects another gene or its product. For example, if a mutation in gene A decreases its ability to phosphorylate protein B, the A allele is annotated to the phenotype “decreased protein kinase activity” with the ID for gene B in an extension. In the downloadable file, an annotation can have multiple comma-separated extensions if they combine to form a “compound” extension (e.g. two or more genes assayed together).</li>
</ul>
</li>
<li><a href="../annotation_properties/">annotation properties</a></li>
<li><a href="../aspect/">aspect</a> - varies by association type<ul>
<li><a href="../gaf_association_aspect/">gaf association➞aspect</a></li>
</ul>
</li>
<li><a href="../assigned_by/">assigned by</a> - assigned by</li>
<li><a href="../associationDocument__associations/">➞associations</a><ul>
<li><a href="../complex_disease_phenotype_ontology_association_document_associations/">complex disease phenotype ontology association document➞associations</a></li>
<li><a href="../gaf_association_document_associations/">gaf association document➞associations</a></li>
<li><a href="../gpad_association_document_associations/">gpad association document➞associations</a></li>
<li><a href="../human_phenotype_ontology_association_document_associations/">human phenotype ontology association document➞associations</a></li>
<li><a href="../pombase_phaf_association_document_associations/">pombase phaf association document➞associations</a></li>
</ul>
</li>
<li><a href="../biocuration/">biocuration</a></li>
<li><a href="../bioentity_type/">bioentity type</a> - type of entity</li>
<li><a href="../citation/">citation</a></li>
<li><a href="../db/">db</a> - db part of identifier<ul>
<li><a href="../Database_name/">Database name</a> - The database that produced the file – included to facilitate data sharing with other databases. Obviously always PomBase for this file.</li>
</ul>
</li>
<li><a href="../db_object_name/">db object name</a></li>
<li><a href="../db_object_ref/">db object ref</a><ul>
<li><a href="../gpi_entity_db_object_ref/">gpi entity➞db object ref</a></li>
</ul>
</li>
<li><a href="../db_object_symbol/">db object symbol</a></li>
<li><a href="../db_object_synonyms/">db object synonyms</a></li>
<li><a href="../db_object_taxon/">db object taxon</a><ul>
<li><a href="../Taxon/">Taxon</a> - The taxon will usually be 4896 (the NCBI taxon ID for Schizosaccharomyces pombe), although an NCBI taxon ID for a specific S. pombe strain would be allowed.</li>
</ul>
</li>
<li><a href="../db_object_type/">db object type</a><ul>
<li><a href="../gaf_association_db_object_type/">gaf association➞db object type</a></li>
<li><a href="../gpad_association_db_object_type/">gpad association➞db object type</a></li>
</ul>
</li>
<li><a href="../db_xrefs/">db xrefs</a></li>
<li><a href="../disease_id/">disease_id</a> - CURIE for </li>
<li><a href="../disease_name/">disease_name</a></li>
<li><a href="../document_metadata_field/">document metadata field</a><ul>
<li><a href="../date_generated/">date generated</a></li>
<li><a href="../funding/">funding</a> - e.g. NHGRI</li>
<li><a href="../generated_by/">generated by</a> - YYYY-MM-DD or YYYY-MM-DDTHH:MM</li>
<li><a href="../ontology_version/">ontology version</a><ul>
<li><a href="../eco_version/">eco version</a></li>
<li><a href="../go_version/">go version</a></li>
<li><a href="../gorel_version/">gorel version</a></li>
<li><a href="../ro_version/">ro version</a></li>
</ul>
</li>
<li><a href="../project_release/">project release</a> - e.g.  WS275</li>
<li><a href="../url/">url</a> - e.g. http://www.yeastgenome.org/</li>
</ul>
</li>
<li><a href="../encoded_by/">encoded by</a></li>
<li><a href="evidence.md">evidence</a></li>
<li><a href="../evidence_type/">evidence type</a><ul>
<li><a href="../Evidence/">Evidence</a> - For the Evidence column, we use a small selection from the Evidence Ontology (ECO). Please contact the Helpdesk if you need assistance using these IDs</li>
</ul>
</li>
<li><a href="../frequency/">frequency</a><ul>
<li><a href="../complex_disease_phenotype_ontology_association_frequency/">complex disease phenotype ontology association➞frequency</a></li>
</ul>
</li>
<li><a href="../gene_product_form/">gene product form</a> - isoform</li>
<li><a href="../gene_product_properties/">gene product properties</a></li>
<li><a href="../gpiDocument__entities/">➞entities</a></li>
<li><a href="../id/">id</a> - identifier</li>
<li><a href="../interacting_taxon_ref/">interacting taxon ref</a></li>
<li><a href="../local_id/">local id</a> - local part of identifier<ul>
<li><a href="../Gene_systematic_ID/">Gene systematic ID</a> - The systematic ID for each gene.</li>
</ul>
</li>
<li><a href="../modifier/">modifier</a></li>
<li><a href="../negation/">negation</a></li>
<li><a href="../onset/">onset</a></li>
<li><a href="../ontology_class_ref/">ontology class ref</a><ul>
<li><a href="../FYPO_ID/">FYPO ID</a> - To use the FYPO IDs, see the <a href="https://www.pombase.org/browse-curation/fission-yeast-phenotype-ontology">FYPO summary page</a> and the FAQ on browsing FYPO.</li>
<li><a href="../gaf_association_ontology_class_ref/">gaf association➞ontology class ref</a> - this must be a GO ID</li>
</ul>
</li>
<li><a href="../parent_protein/">parent protein</a></li>
<li><a href="../propertyValuePair__property/">➞property</a></li>
<li><a href="../propertyValuePair__value/">➞value</a> - value</li>
<li><a href="../protein_containing_complex_members/">protein containing complex members</a></li>
<li><a href="../qualifier/">qualifier</a></li>
<li><a href="../qualifiers/">qualifiers</a><ul>
<li><a href="../gaf_association_qualifiers/">gaf association➞qualifiers</a><ul>
<li><a href="../simple_gaf_association_qualifiers/">simple gaf association➞qualifiers</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="relation.md">relation</a><ul>
<li><a href="../complex_disease_phenotype_ontology_association_relation/">complex disease phenotype ontology association➞relation</a></li>
<li><a href="../gpad_association_relation/">gpad association➞relation</a></li>
</ul>
</li>
<li><a href="../sex/">sex</a></li>
<li><a href="../supporting_references/">supporting references</a><ul>
<li><a href="Reference.md">Reference</a> - The Reference column has the publication’s PubMed ID (PMID).</li>
</ul>
</li>
<li><a href="../term_id/">term_id</a></li>
<li><a href="../term_label/">term_label</a></li>
<li><a href="../time_course/">time_course</a></li>
<li><a href="../with_or_from/">with or from</a></li>
</ul>
<h3 id="enums">Enums</h3>
<ul>
<li><a href="../Allele_type_enum/">Allele_type_enum</a></li>
<li><a href="../Biological_Effect_interactor_enum/">Biological_Effect_interactor_enum</a></li>
<li><a href="../Biological_role_interactor_enum/">Biological_role_interactor_enum</a></li>
<li><a href="../Causal_Regulatory_Mechanism_enum/">Causal_Regulatory_Mechanism_enum</a></li>
<li><a href="../ComplexDiseaseRelation/">ComplexDiseaseRelation</a></li>
<li><a href="../Database_name_enum/">Database_name_enum</a></li>
<li><a href="../Evidence_enum/">Evidence_enum</a></li>
<li><a href="../Expansion_method_enum/">Expansion_method_enum</a></li>
<li><a href="../Experimental_role_interactor_enum/">Experimental_role_interactor_enum</a></li>
<li><a href="../Expression_enum/">Expression_enum</a></li>
<li><a href="../Interaction_Xref_enum/">Interaction_Xref_enum</a></li>
<li><a href="../Interaction_detection_method_enum/">Interaction_detection_method_enum</a></li>
<li><a href="../Interaction_type_enum/">Interaction_type_enum</a></li>
<li><a href="../Negative_enum/">Negative_enum</a></li>
<li><a href="../Stoichiometry_interactor_enum/">Stoichiometry_interactor_enum</a></li>
<li><a href="../gene_ontology_aspect_enum/">gene_ontology_aspect_enum</a></li>
<li><a href="../gp2term_relation_enum/">gp2term_relation_enum</a></li>
<li><a href="../gp_entity_type_enum/">gp_entity_type_enum</a></li>
<li><a href="../simple_qualifier_enum/">simple_qualifier_enum</a></li>
</ul>
<h3 id="subsets">Subsets</h3>
<h3 id="types">Types</h3>
<h4 id="built-in">Built in</h4>
<ul>
<li><strong>Bool</strong></li>
<li><strong>Decimal</strong></li>
<li><strong>ElementIdentifier</strong></li>
<li><strong>NCName</strong></li>
<li><strong>NodeIdentifier</strong></li>
<li><strong>URI</strong></li>
<li><strong>URIorCURIE</strong></li>
<li><strong>XSDDate</strong></li>
<li><strong>XSDDateTime</strong></li>
<li><strong>XSDTime</strong></li>
<li><strong>float</strong></li>
<li><strong>int</strong></li>
<li><strong>str</strong></li>
</ul>
<h4 id="defined">Defined</h4>
<ul>
<li><a href="../types/FYPOIdentifier/">FYPOIdentifier</a>  (<a href="../types/String/">String</a>) </li>
<li><a href="../types/FYPOEXTIdentifier/">FYPOEXTIdentifier</a>  (<a href="../types/String/">String</a>) </li>
<li><a href="../types/HPOIdentifier/">HPOIdentifier</a>  (<a href="../types/IdentifierType/">IdentifierType</a>) </li>
<li><a href="../types/MONDOIdentifier/">MONDOIdentifier</a>  (<a href="../types/IdentifierType/">IdentifierType</a>) </li>
<li><a href="../types/PECOIdentifier/">PECOIdentifier</a>  (<a href="../types/String/">String</a>) </li>
<li><a href="../types/PMIDIdentifier/">PMIDIdentifier</a>  (<a href="../types/IdentifierType/">IdentifierType</a>) </li>
<li><a href="../types/Boolean/">Boolean</a>  (<strong>Bool</strong>)  - A binary (true or false) value</li>
<li><a href="../types/Curie/">Curie</a>  (<a href="../types/String/">String</a>) </li>
<li><a href="../types/Date/">Date</a>  (<strong>XSDDate</strong>)  - a date (year, month and day) in an idealized calendar</li>
<li><a href="../types/Datetime/">Datetime</a>  (<strong>XSDDateTime</strong>)  - The combination of a date and time</li>
<li><a href="../types/Decimal/">Decimal</a>  (<strong>Decimal</strong>)  - A real number with arbitrary precision that conforms to the xsd:decimal specification</li>
<li><a href="../types/Double/">Double</a>  (<strong>float</strong>)  - A real number that conforms to the xsd:double specification</li>
<li><a href="../types/Float/">Float</a>  (<strong>float</strong>)  - A real number that conforms to the xsd:float specification</li>
<li><a href="../types/Identifier/">Identifier</a>  (<a href="../types/String/">String</a>) </li>
<li><a href="../types/IdentifierType/">IdentifierType</a>  (<a href="../types/String/">String</a>) </li>
<li><a href="../types/Intact-miscoreIdentifier/">Intact-miscoreIdentifier</a>  (<a href="../types/String/">String</a>) </li>
<li><a href="../types/Integer/">Integer</a>  (<strong>int</strong>)  - An integer</li>
<li><a href="../types/NameType/">NameType</a>  (<a href="../types/String/">String</a>) </li>
<li><a href="../types/Ncname/">Ncname</a>  (<strong>NCName</strong>)  - Prefix part of CURIE</li>
<li><a href="../types/Nodeidentifier/">Nodeidentifier</a>  (<strong>NodeIdentifier</strong>)  - A URI, CURIE or BNODE that represents a node in a model.</li>
<li><a href="../types/Objectidentifier/">Objectidentifier</a>  (<strong>ElementIdentifier</strong>)  - A URI or CURIE that represents an object in the model.</li>
<li><a href="../types/PhenotypeIdentifier/">PhenotypeIdentifier</a>  (<a href="../types/IdentifierType/">IdentifierType</a>) </li>
<li><a href="../types/Psi-miIdentifier/">Psi-miIdentifier</a>  (<a href="../types/String/">String</a>) </li>
<li><a href="../types/RogidIdentifier/">RogidIdentifier</a>  (<a href="../types/String/">String</a>) </li>
<li><a href="../types/String/">String</a>  (<strong>str</strong>)  - A character string</li>
<li><a href="../types/SymbolType/">SymbolType</a>  (<a href="../types/String/">String</a>) </li>
<li><a href="../types/TaxidIdentifier/">TaxidIdentifier</a>  (<a href="../types/String/">String</a>) </li>
<li><a href="../types/Time/">Time</a>  (<strong>XSDTime</strong>)  - A time object represents a (local) time of day, independent of any particular day</li>
<li><a href="../types/Uri/">Uri</a>  (<strong>URI</strong>)  - a complete URI</li>
<li><a href="../types/Uriorcurie/">Uriorcurie</a>  (<strong>URIorCURIE</strong>)  - a URI or a CURIE</li>
</ul>
              
            </div>
          </div>
          <footer>
  

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
    
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
      
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="versions">
  <span class="rst-current-version" data-toggle="rst-current-version">
    
        <span>
          <a href="https://github.com/biodatamodels/ontology-associations/" class="fa fa-github" style="color: #fcfcfc"> GitHub</a>
        </span>
    
    
    
  </span>
</div>
    <script>var base_url = '..';</script>
    <script src="../js/theme_extra.js" defer></script>
    <script src="../js/theme.js" defer></script>
      <script src="../search/main.js" defer></script>
    <script defer>
        window.onload = function () {
            SphinxRtdTheme.Navigation.enable(true);
        };
    </script>

</body>
</html>
